packages/blade/src/utils/storybook/StoryPageWrapper.tsx

```diff
@@ -116,6 +125,22 @@ const StoryPageWrapper = (props: StoryPageWrapperTypes): React.ReactElement => {
             <>
               <BaseBox id="properties-ref">
                 <Title>Properties</Title>
+                {props.apiDecisionLink === '' ? null : (
+                  <Text marginY="spacing.5">
+                    You can checkout API Decisions for {props.componentName} at{' '}
+                    <Link
+                      target="_blank"
+                      href={
+                        props.apiDecisionLink ??
+                        `https://github.com/razorpay/blade/blob/master/packages/blade/src/components/${props.componentName}/_decisions/decisions.md`
```

Comment:
So, by default we use this link, unless explicitly mentioned via prop. In case it doesn't have apiDescision we need to pass an empty string `apiDecisionLink=''`. Is that right?

Reply:
yup correct!